BREAKING.md

```diff
@@ -19,21 +19,22 @@ It's important to communicate breaking changes to our stakeholders. To write a g
 
 ## 2.0.0-internal.4.1.0 Upcoming changes
 
--   [garbage-collector and related items deprecated](#garbage-collector-and-related-items-deprecated)
+-   [@fluidframework/garbage-collector deprecated](#@fluidframework/garbage-collector-deprecated)
```

Comment:
@tylerbutler - You would want to add changesets for this. Also, is it fine to update an existing note since we haven't released internal.4.1 yet?

Reply:
I'm adding a changeset; and yes, the contents of the changesets files can be edited any time. When we release, they'll be deleted.

Reply:
I just updated the existing changeset.